tests/core/iio/task.yaml

```diff
@@ -9,7 +9,7 @@ details: |
     that, after a snap declared a iio plug is installed and connected, it can
     access the node and read/write its content.
 
-systems: [ubuntu-core-16-64]
+systems: [ubuntu-core-*-64]
```

Comment:
This test seems to be failing on all the other UC systems - this was due to IIO missing right?

Reply:
right, I see this error: 
error: snap "snapd" has no slot named "iio0"

@Meulengracht  @alfonsosanchezbeato, What do you suggest? Should we remove the test? should we add that interface to snapd?

Reply:
If it was intentional the interface does not exist in the snapd snap, but exists on the core snap, then this must exist just as an UC16 test, and that means we should remove it from `master`. As long as this runs on the branch of snapd for UC16, then its good enough.

If the interface is missing from snapd, then we need to keep this test and fix that